multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madtheme/MadTheme.kt

```diff
@@ -150,6 +161,37 @@ abstract class MadTheme(
     }
 
     // Chapters
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        // API is heavily rate limited. Use custom client
+        return if (manga.status != SManga.LICENSED) {
+            chapterClient.newCall(chapterListRequest(manga))
+                .asObservable()
+                .map { response ->
+                    chapterListParse(response)
+                }
+        } else {
+            Observable.error(Exception("Licensed - No chapters to show"))
+        }
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        if (response.code in 200..299) {
+            return super.chapterListParse(response)
+        }
+
+        // Try to use message from site
+        response.body?.let { body ->
+            val message: String? = json
+                .decodeFromString<JsonObject>(body.string())["message"]
+                ?.jsonPrimitive
+                ?.toString()
```

Comment:
Using `toString()` will return the JSON representation and not the actual value IIRC.

```suggestion
                ?.content
```

Reply:
`.toString()` returns it with surrounding quotes, `.content` is correct